api/v1alpha1/dataset_types.go

```diff
@@ -353,7 +353,7 @@ func (dataset *Dataset) SetDataOperationInProgress(operationType string, name st
 		}
 		return
 	}
-	if dataset.Status.OperationRef[operationType] == "" {
+	if dataset.Status.OperationRef[operationType] == "" || strings.Contains(dataset.Status.OperationRef[operationType], name) {
```

Comment:
If "A,B" contains "A" , the result should be "A,B" not "A".

Reply:
fixed